src/main/java/core/basesyntax/dao/ConverterDataToObjImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.FruitTransaction;
+import java.util.List;
+
+public class ConverterDataToObjImpl {
+    private static final String SPLITERATOR = ",";
+    private static final int OPERATION_SEQUENCE = 0;
+    private static final int FRUIT_NAME_SEQUENCE = 1;
+    private static final int FRUIT_QUANTITY_SEQUENCE = 2;
+
+    public List<FruitTransaction> convertData(String filePath,
+                                              List<String> listOfStrings) {
```

Comment:
1. you dont need that argument
2. unclear naming
```suggestion
    public List<FruitTransaction> convertData(List<String> inputFruitsData) {
```